packages/typespec-vscode/src/vscode-cmd/emit-code/emit-code.ts

```diff
@@ -261,6 +265,17 @@ async function doEmit(
     },
   );
 
+  if (installPackageQuickPickItemsResult.code !== ResultCode.Success) {
+    logger.error("Failed to calculate packages to install. Emitting cancelled by user.", [], {
```

Comment:
it's not necessary 'Cancelled". It can also be "Failed" or "Timeout". same for other places